src/frontend-samples/explode-sample/ExplodeApp.tsx

```diff
@@ -0,0 +1,337 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as React from "react";
+import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
+import "common/samples-common.scss";
+import SampleApp from "common/SampleApp";
+import ExplodeUI from "./ExplodeUI";
+import { CoordSystem, DecorateContext, Decorator, EmphasizeElements, FeatureOverrideProvider, FeatureSymbology, GraphicBranch, GraphicType, ImdlReader, IModelApp, SceneContext, ScreenViewport, TileContent, TiledGraphicsProvider, TileRequest, TileTreeReference, Viewport } from "@bentley/imodeljs-frontend";
+import { Matrix3d, Point3d, Point4d, Range3d, Transform, Vector3d } from "@bentley/geometry-core";
+import { ElementGraphicsRequestProps, FeatureAppearance, IModelTileRpcInterface, Placement3d, TileVersionInfo } from "@bentley/imodeljs-common";
+import { ByteStream } from "@bentley/bentleyjs-core";
+
+interface ElementData {
```

Comment:
This should be split into two interfaces: ElementDataProps to represent the data you get back from your query (and replace Point3d with XYZProps - queries do not return instances of classes); and ElementData to hold the data you create from the query results.

Reply:
please do not resolve other people's comments on your PRs.